### DISCUSSION TITLE
Release a package compatible with ARM processors architecture

### DISCUSSION THREAD

**Discussion by fsidhoum** [massalabs/massa#2812] [Ideas]
Hello the Massa team,

It would be great if, when you release a new version, a package with ARM architecture compatible binaries is also generated.

This is not a blocking issue because ARM processor users can build the binaries directly from the sources but it would be more convenient if the binaries would be included in the release.

**Comment by qdrn** [comment]
Which platform are you using ?

**Reply by fsidhoum** [reply]
I am using a Ubuntu VM with an ARM Ampere processor.
